extensions/sql-migration/src/wizard/skuRecommendation/assessmentSummaryCard.ts

```diff
@@ -386,18 +386,20 @@ export class AssessmentSummaryCard implements vscode.Disposable {
 	}
 
 	// Used to update the summary of assessment card later after its initialization.
-	// TODO - We can remove dummayData input later and use a some other way to pass the values. Once we start implementing the whole page.
-	public async updateContent(dummyData: DummyData) {
-		await this._assessmentResultText.updateProperties({ "value": constants.ASSESSED_DBS(dummyData.assessmentResult) });
-		await this._readyText.updateProperties({ "value": dummyData.ready.toString() });
-		await this._needsReviewText.updateProperties({ "value": dummyData.needsReview.toString() });
-		await this._notReadyText.updateProperties({ "value": dummyData.notReady.toString() });
-		await this._blockersText.updateProperties({ "value": dummyData.blockers.toString() });
-		await this._warningsText.updateProperties({ "value": dummyData.warnings.toString() });
-
-		await this._recommendedConfigurationText.updateProperties({ "value": dummyData.skuRecommendation.toString() });
-		await this._vmRecommendedConfigurationText.updateProperties({ "value": dummyData.vmRecommendation });
+	public updateAssessmentResult(
+		databaseCounts: number, ready: number, needsReview: number, notReady: number, blockers: number, warnings: number) {
+		this._assessmentResultText.value = constants.ASSESSED_DBS(databaseCounts);
+		this._readyText.value = ready.toString();
+		this._needsReviewText.value = needsReview.toString();
+		this._notReadyText.value = notReady.toString();
+		this._blockersText.value = blockers.toString();
+		this._warningsText.value = warnings.toString();
+	}
 
+	public async updateSkuRecommendation(
+		skuRecommendation: string, vmRecommendation: string) {
+		await this._recommendedConfigurationText.updateProperties({ "value": skuRecommendation });
```

Comment:
Let's make these two as well this._recommendedConfigurationText.updateProperties.value = skurecommendation;
this way we can remove unnecessary async from this function.

Reply:
Thanks @stuti149 for pointing this out. I was thinking I can do for this Function in next PR as well. Since, this function might require some signature changes as well as per requirement.